bazel/external/dpdk.BUILD

```diff
@@ -25,8 +25,7 @@ cc_library(
         "lib/libtdi_pna.so*",
         "lib/libtdi.so*",
         # DPDK libs
-        "lib/x86_64-linux-gnu/dpdk/pmds-22.2/*.so*",
-        "lib/x86_64-linux-gnu/librte_*.so*",
+        "lib/x86_64-linux-gnu/**/*.so*",
```

Comment:
The latest DPDK uses `pmds-23.0` - this way we can be independent from the DPDK version.

Reply:
We generally careful what we package, as the files here get installed under `/usr/lib` and `apt` does not allow conflicts. Could this below also fix it?

```suggestion
        "lib/x86_64-linux-gnu/pmds-/*.so*",
        "lib/x86_64-linux-gnu/librte_*.so*",
```